libs/blocks/merch/merch.js

```diff
@@ -496,7 +497,13 @@ export async function openModal(e, url, offerType, hash, extraOptions) {
     const fragmentPath = url.split(/(hlx|aem).(page|live)/).pop();
     modal = await openFragmentModal(fragmentPath, getModal);
   } else {
-    modal = await openExternalModal(url, getModal, extraOptions);
+    const isThreeInOneModal = [MODAL_TYPE_3_IN_1.CRM, MODAL_TYPE_3_IN_1.D2P, MODAL_TYPE_3_IN_1.TWP].includes(el?.getAttribute('data-modal-type')) && el?.href;
```

Comment:
If we add additional modal types we would need to maintain the code in two places. Why not use `Object.values` to create an array from the values?